schemas/IDLs/org.gel.models.cva.avro/1.5.0/DataIntake.avdl

```diff
@@ -0,0 +1,318 @@
+@namespace("org.gel.models.cva.avro")
+
+/**
+The DataIntakeProtocol defines two records for injecting report events into CVA:
+
+   - InterpretedGenomeInject
+   - ClinicalReportInject
+
+The above entities contain either an InterpretedGenome or a ClinicalReport that have list of variants,
+having themselves lists of report events. The variants will go through the variant ingestion pipeline,
+normalised, checked against the reference genome for validity and lifted over.
+
+The DataIntakeProtocol defined two records to inject exit questionnaires into CVA:
+
+    - ExitQuestionnaireInjectRD
+    - ExitQuestionnaireInjectCancer
+
+The exit questionnaires are transformed into a report event for storage in CVA. Variants are also
+processed through the variant ingestion pipeline as described above.
+
+The DataIntakeProtocol defines two records for injecting sample information into CVA:
+
+    - PedigreeInject
+    - CancerParticipantInject
+
+Pedigree and cancer participant data are stored in ad hoc collections and the most relevant bits of
+information (e.g.: proband present HPO terms) are used to annotate the cases.
+
+When any of these entities is pushed to CVA, a Transaction will be created and processing will happen
+asynchronously. Successful push does not imply successful processing.
+
+*/
+protocol DataIntakeProtocol {
+
+    import idl "CvaVariant.avdl";
+    import idl "CvaEvidence.avdl";
+    import idl "InterpretedGenome.avdl";
+    import idl "ClinicalReport.avdl";
+    import idl "ExitQuestionnaire.avdl";
+    import idl "RDParticipant.avdl";
+    import idl "CancerParticipant.avdl";
+    import idl "Ngis.avdl";
```

Comment:
The build currently fails because you are missing an import here. I think you need 
`import idl VariantInterpretationLog.avdl`

Reply:
Thanks. Fixed